docs/api-docs/payments/payments-api-overview.md

```diff
@@ -403,79 +287,88 @@ When processing a credit payment, set `save_instrument: true`. The shopper can a
       "cardholder_name": "BP",
       "expiry_month": 12,
       "expiry_year": 2020,
-      "verification_value": "411"
+      "verification_value": "411" // card CVV/CVC
     },
     "payment_method_id": "authorizenet.card",
     "save_instrument": true
   }
 }
 ```
----
+When processing a payment using a paypal account, set thet `type` to `stored_paypal_account`. 
 
-<a href='#payments_orders-api' aria-hidden='true' class='block-anchor'  id='payments_orders-api'><i aria-hidden='true' class='linkify icon'></i></a>
+```http title="Example request: Process payment and save paypal account" lineNumbers
+POST https://payments.bigcommerce.com/stores/{{store_hash}}/payments
+Accept: application/vnd.bc.v1+json
+Content-Type: application/json
+
+{
+  "payment": {
+    "instrument": {
+      "type": "stored_paypal_account",
+      "token": {{token}}
+    },
+    "payment_method_id": "braintree.paypal"
+  }
+}
+```
```

Comment:
This section should move closer to [`paying via stored payment instrument` section](https://github.com/bigcommerce/dev-docs/pull/1398/files#diff-bc715d93134c9b6d37fa932cd73ac23484afc4b6cdf89df5a84a013c07c87e05R165)
At the moment, it is under Storing payment instrument section.